libipl/p10/common.C

```diff
@@ -232,3 +231,59 @@ void ipl_log_sbe_ffdc(struct pdbg_target *pib)
 	if (ffdc)
 		free(ffdc);
 }
+
+int ipl_set_sbe_state(struct pdbg_target *proc, enum sbe_state state)
+{
+	ipl_log(IPL_INFO, "(%s)SBE state update (%d) \n",
+		pdbg_target_path(proc), state);
+
+	// get PIB target
+	char path[16];
+	sprintf(path, "/proc%d/pib", pdbg_target_index(proc));
+	struct pdbg_target *pib = pdbg_target_from_path(nullptr, path);
+	if (pib == nullptr) {
+		ipl_log(IPL_ERROR, "Failed to get PIB target for(%s)",
+			pdbg_target_path(proc));
+		ipl_error_callback(IPL_ERR_PIB_TGT_NOT_FOUND);
+		return 1;
+	}
+	//PIB already probed as part of ipl init function.
+	//update SBE state
+	if (sbe_set_state(pib, state)) {
+		ipl_log(IPL_ERROR, "Failed to update SBE state information (%s)",
+			pdbg_target_path(proc));
+		ipl_error_callback(IPL_ERR_CFAM);
+		return 1;
+	}
+	return 0;
+}
+
+int ipl_set_sbe_state_all(enum sbe_state state)
+{
+	struct pdbg_target *proc;
+	int ret = 0;
+	pdbg_for_each_class_target("proc", proc) {
+		if (ipl_is_present(proc)) {
+			if (ipl_set_sbe_state(proc, state)) {
```

Comment:
this method does not indicate it is handling failure case, first look i thought if success set ret =1. probably some comment or
if (... == FAILURE)
{
   ret = FAILURE;
}


Probabuly some enums or constants would be good for readability

Reply:
general convention follwed here , 0 for success and non zero (1) for failure. added details in function description

Reply:
* @return 0 on success, 1 on failure

Reply:
k